packages/components/src/drop-zone/provider.js

```diff
@@ -143,10 +143,15 @@ class DropZoneProvider extends Component {
 		if ( hoveredDropZone ) {
 			const rect = hoveredDropZone.element.current.getBoundingClientRect();
 
-			position = {
-				x: detail.clientX - rect.left < rect.right - detail.clientX ? 'left' : 'right',
-				y: detail.clientY - rect.top < rect.bottom - detail.clientY ? 'top' : 'bottom',
-			};
+			let x = detail.clientX;
+			let y = detail.clientY;
+
+			if ( ! hoveredDropZone.withExactPosition ) {
```

Comment:
should we just use another property like "coordinates" instead of hacking two different types in the same property?

Reply:
I personally wouldn't, because it will make the component rerender much more, but I'm fine with it either way. We could remove the top/bottom calculation since they were specific for blocks.

Reply:
Although `is-close-to-top` is part of `DropZone`'s API. Wouldn't  mind removing it if it's ok, nothing in Gutenberg is now using these classes.

Reply:
I don't mind dropping it too if not used. (dev note)

Reply:
Ok, I removed them. :)